socorro/lib/search_common.py

```diff
@@ -140,6 +136,11 @@ def build_filters(self, fields):
     def get_parameters(self, **kwargs):
         parameters = {}
 
+        fields = kwargs.get('_fields')
```

Comment:
`fields = kwargs['_fields']` because it's not an option that this falls back on `None`. 
Keep the `assert` on the next line though.